patient-care/webapp/.meteor/versions

```diff
@@ -73,7 +73,7 @@ less@2.6.5
 livedata@1.0.18
 localstorage@1.0.11
 logging@1.0.14
-matb33:collection-hooks@0.8.1
+matb33:collection-hooks@0.8.4
```

Comment:
This might actually be why the patient-care build is taking forever? (This would cause the Travis build to take too long and timeout.)